editor/editor_node.cpp

```diff
@@ -484,8 +484,8 @@ void EditorNode::_update_from_settings() {
 	RS::get_singleton()->environment_set_sdfgi_frames_to_converge(frames_to_converge);
 	RS::EnvironmentSDFGIRayCount ray_count = RS::EnvironmentSDFGIRayCount(int(GLOBAL_GET("rendering/global_illumination/sdfgi/probe_ray_count")));
 	RS::get_singleton()->environment_set_sdfgi_ray_count(ray_count);
-	RS::GIProbeQuality gi_probe_quality = RS::GIProbeQuality(int(GLOBAL_GET("rendering/global_illumination/gi_probes/quality")));
-	RS::get_singleton()->gi_probe_set_quality(gi_probe_quality);
+	RS::VoxelGIQuality voxel_gi_quality = RS::VoxelGIQuality(int(GLOBAL_GET("rendering/global_illumination/voxel_gis/quality")));
```

Comment:
```suggestion
	RS::VoxelGIQuality voxel_gi_quality = RS::VoxelGIQuality(int(GLOBAL_GET("rendering/global_illumination/voxel_gi/quality")));
```

This one might be an issue throughout the engine. It looks like you search and replaced "gi_probe" with "voxel_gi" however, at some places we had "gi_probes". So now you need to find and replace "voxel_gis" with "voxel_gi"

Reply:
Good catch. I will rename voxel_gis to voxel_gi_instances and in the configuration, voxel_gis to voxel_gi.